zenoh-flow/src/runtime/resources.rs

```diff
@@ -308,18 +315,18 @@ impl DataStore {
     }
 
     pub async fn get_all_runtime_info(&self) -> ZFResult<Vec<RuntimeInfo>> {
-        let selector = zenoh::Selector::try_from(RT_INFO_PATH!(ROOT_STANDALONE, "*"))?;
-        let ws = self.z.workspace(None).await?;
-        let mut ds = ws.get(&selector).await?;
+        let selector = RT_INFO_PATH!(ROOT_STANDALONE, "*");
+
+        let mut ds = self.z.get(&selector).await?;
 
         // Not sure this is needed...
```

Comment:
How can we know if it is indeed needed?

Reply:
You mean the `mut` on the variable `ds`?

Reply:
I’m referring to the comment: `// Not sure this is needed…` (which appears regularly). So most likely the line under `let data = ds.collect::Vec<Reply>>().await;`. Do we have a way of knowing if it’s needed?

Reply:
Removing the comment :D